src/main/java/com/onevizion/scmdb/SqlScriptExecutor.java

```diff
@@ -71,28 +71,29 @@ public int execute(SqlScript script) {
             ScriptRunnerContext ctx = new ScriptRunnerContext();
 
             ctx.setBaseConnection(connection);
+            ByteArrayOutputStream baos = new ByteArrayOutputStream();
+            BufferedOutputStream bos = new BufferedOutputStream(baos);
+            ctx.setOutputStreamWrapper(bos);
             executor.setScriptRunnerContext(ctx);
             executor.setStmt(String.format(SQL_COMMAND, wrapperScriptFile.getAbsolutePath(),
-                                           script.getFile().getAbsolutePath()));
+                    script.getFile().getAbsolutePath()));
 
             Instant start = Instant.now();
-
-            ByteArrayOutputStream baos = new ByteArrayOutputStream();
-            PrintStream ps = new PrintStream(baos);
-            PrintStream old = System.out;
-            System.setOut(ps);
+            executor.setOut(ctx.getOutputStream());
             executor.run();
-            System.out.flush();
-            System.setOut(old);
             script.setOutput(baos.toString());
             System.out.println(baos.toString());
```

Comment:
the script output should be written to the console as available, not when script execution is finished, find a way to make `executor` write into 2 streams: `system.out` and stream for saving into db

Reply:
done